packages/terra-search-field/src/SearchField.module.scss

```diff
@@ -50,8 +50,8 @@
       flex: 0 0 auto;
       margin: 0;
       position: absolute;
-      right: var(-terra-search-field-clear-right, 0);
-      top: var(-terra-search-field-clear-top, 0);
+      right: var(--terra-search-field-clear-right, 0);
+      top: var(--terra-search-field-clear-top, 0);
```

Comment:
do our themes actually use these variables? if they did, i'd expect us to update screenshots. i did a theme-wide search but got back 0 matches 😕perhaps we should remove these custom css variables instead?

Reply:
unless it's used by consumer or clinical... if we can't remove, we should add them to the orion-fusion-theme & clinical-lowlight-theme since we are trying to back-fill the partial themes

Reply:
I agree with @emilyrohrbough

Reply:
yeah, when i mentioned theme-wide search, that also included clinical and consumer themes :D unless i missed it, they weren't being used there

Reply:
Sounds like these can be removed based on Derek's search & usage.

Reply:
Removed d65e1ce